src/components/asthma/components/AsthmaLibraryArticles/AsthmaLibraryArticles.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './AsthmaLibraryArticles.css';
+import { AsthmaLibraryArticle } from '../../model';
+import MyDataHelps from '@careevolution/mydatahelps-js';
+import { Resource, ResourceImageAlignment } from '../../../presentational';
+import { AsthmaLibraryArticlesPreviewState, previewData } from './AsthmaLibraryArticles.previewData';
+import language from '../../../../helpers/language';
+
+export interface AsthmaLibraryArticlesProps {
+    previewState?: AsthmaLibraryArticlesPreviewState;
+    articles: AsthmaLibraryArticle[];
+    imageAlignment?: ResourceImageAlignment;
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function (props: AsthmaLibraryArticlesProps) {
+    const onClick = (article: AsthmaLibraryArticle): void => {
+        if (props.previewState) return;
+        MyDataHelps.openEmbeddedUrl(article.url);
+    };
+
+    let articles = props.previewState ? previewData[props.previewState].articles : props.articles;
+
+    return <div className="mdhui-asthma-library-articles" ref={props.innerRef}>
+        <>
```

Comment:
Decoupling opportunity.
This is resource list

Reply:
Agreed^ nothing Asthma specific here that I can see?

Reply:
I have move this out to the common component area and renamed it `ResourceList`.